src/metatrain/utils/data/readers/ase.py

```diff
@@ -182,3 +189,162 @@ def _read_virial_stress_ase(
         blocks.append(block)
 
     return blocks
+
+
+def read_energy(target: DictConfig) -> Tuple[List[TensorMap], TargetInfo]:
```

Comment:
How does this differ from `read_energy_ase` and why are there now functions `read_energy_ase` and `read_energy` in this file?

it seems like the `read_xxx_ase` are not used publically anymore and the only important functions are `read_systems`, `read_energy`, `read_generic`. If this is true you can make these functions private to be less confusing.

Reply:
Sure. It turns out that reading energy, forces and stress separately is reader-specific. For example, if you read `TensorMaps`, you read everything at once including the gradient blocks